src/main/java/uk/gov/hmcts/darts/authentication/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.hmcts.darts.authentication.service.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.Response;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.utils.URIBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.util.StreamUtils;
+import uk.gov.hmcts.darts.authentication.config.AuthenticationConfiguration;
+import uk.gov.hmcts.darts.authentication.model.OAuthProviderRawResponse;
+import uk.gov.hmcts.darts.authentication.service.AuthenticationService;
+import uk.gov.hmcts.darts.authentication.service.AzureActiveDirectoryB2CClient;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+@Slf4j
```

Comment:
Is this the correct import annotation? Looking at https://projectlombok.org/api/lombok/extern/slf4j/Slf4j
It sounds like line 27 isn't needed - It causes lombok to generate a logger field.

Reply:
Good catch. I think this groovy Slf4j was added with a manual logger at the very beginning of the development. Its cleaned now.